### ISSUE TITLE
New-Vesterconfig cannot differentiate datacenter with similar names

### ISSUE THREAD

**Issue by GodKratos** [WahlNetwork/Vester#167]
## Expected Behavior
Selecting the data center selects the correct object

## Current Behavior
Two datacenter objects exist with similar names:
1. Britomart Development
2. Britomart

Selecting either of them chooses the first one only so the second one is unable to be selected.

## Possible Solution
Name selection needs to be an exact match instead of a "like" match

## Steps to Reproduce (for bugs)
In the screesnhots below the only difference between them is selecting a different datacenter in each but the cluster and list of hosts is the same for both (ie "Britomart Development" datacenter has been selected for both)
https://www.dropbox.com/s/f9f8bq816noqm4i/vester1.jpg?dl=1
https://www.dropbox.com/s/imaqfshla0w1312/vester2.jpg?dl=1

## Context
Currently unable to select the correct datacenter to run the script on

## Your Environment
* PSVersion: 5.1.14409.1005
* OSVersion: Microsoft Windows NT 6.3.9600.0
* Pester: 4.0.5
* Vester: 1.1.0
* VMware.PowerCLI: 6.5.1.5377412
* VMware.VimAutomation.Cis.Core: 6.5.1.5374323

**Comment by GodKratos** [comment]
Actually... my expectation appears to be wrong, I assumed the subsequent selections were based off the previous selected options but don't seem to be. Actual issue appears to be with running { $Cluster    = $Datacenter | Get-Cluster -Name $config.scope.cluster } which is giving an error from the script.

When I run { get-cluster -Name '*' }  that works without an error.

**Comment by GodKratos** [comment]
Problem seems to be something strange happening when piping the $Datacenter array to the get-cluster command. If I pipe the array (containing two datacenter objects) to the get-cluster command one of them fails to execute but if I pipe the command from each of the array objects (eg $Datacenter[0]) then the commands work fine.

https://www.dropbox.com/s/71pwckh2j3vl2ou/vester3.jpg?dl=1

**Comment by brianbunke** [comment]
Welcome, @GodKratos!

So that's fun. It looks like `Get-Cluster` is incorrectly binding the `[DatacenterImpl]` objects to the `-VMHost` parameter, instead of `-Location`.

I'll hit up the PowerCLI team and see if they know about it.

In the meantime, if needed, we can compensate for that:

```posh
# Old
Get-Datacenter | Get-Cluster
# New
Get-Cluster -Location (Get-Datacenter)
```

**Comment by GodKratos** [comment]
Thanks, changing that line above does indeed work!

Not sure if it is binding to the VMHost parameter though because if I run Get-Cluster -VMHost (Get-Datacenter) I get quite a different error.

**Comment by brianbunke** [comment]
Ah, that's fair. I was assuming based on the error message, but didn't confirm. Interesting.

**Comment by brianbunke** [comment]
This is confirmed as a PowerCLI bug, and it won't be resolved in PowerCLI `6.5.2`.

I'm thinking we can change Vester to officially use `Get-Cluster -Location (Get-Datacenter)` in the interim. Any interest in submitting a Pull Request? If not, I can tackle soon.

Thanks again for reporting!

**Comment by brianbunke** [comment]
Resolved in #180 🎉
